tests/test_object.py

```diff
@@ -4,7 +4,7 @@
 from bson import BSONCoding, dumps, loads, import_class
 
 
-class TestData(BSONCoding):
+class _TestData(BSONCoding):
```

Comment:
Eliminates a warning from pytest.

Reply:
Should be enough to add `__test__ = False`.

<img width="770" alt="Screenshot 2024-12-18 at 11 04 40" src="https://github.com/user-attachments/assets/162f37af-0948-4d94-a339-d02de1c0d5f4" />

```suggestion
class TestData(BSONCoding):
    __test__ = False
```

Reply:
Yes, it works, thank you, I learnt about this feature.